client/devdocs/style.scss

```diff
@@ -140,6 +140,31 @@
 	margin-top: 48px;
 }
 
+.design-assets {
+	@include breakpoint( "<660px" ) {
+		padding: 0 6px;
+	}
+}
+
+.design-assets h2,
```

Comment:
actually, no, I think it's the rules higher up that are only scoped to `.devdocs__doc`.